app/components/NoticeTypeCheckboxes/rates.ts

```diff
@@ -274,4 +274,5 @@ export const triggerRate: Record<string, number> = {
   'gcn.notices.swift.bat.guano': 0.0,
   'igwn.gwalert': 90.42857142857143,
   'swift.bat.guano': 0.0,
+  'gcn.notices.einstein_probe.wxt.alert': 0.0,
```

Comment:
On mission page, 100 alerts are expected per year, thus rate value is modified.
```suggestion
  'gcn.notices.einstein_probe.wxt.alert': 0.27,
```